### ISSUE TITLE
LaTeXTools Instructions

### ISSUE THREAD

**Issue by ig0774** [andrewheiss/SublimeKnitr#38]
Hi! Just to let you know, the patching instructions you have for LaTeXTools are quite a bit out of date. These days, to use `.rnw` files with LaTeXTools, you'd just have to tweak one setting in the LaTeXTools settings file called `tex_file_exts` and change it from the default value of `[".tex"]` to `[".tex", ".rnw"]`.

**Comment by ernestyao** [comment]
But you still have to modify Packages/LaTeXTools/makePDF.py.

**Comment by ig0774** [comment]
I wouldn't recommend it... The script builder feature or even a custom builder pile be used to run the necessary command in a way that wouldn't be overwritten every time LaTeXTools is updated (especially since the lines referenced in the docs here have changed a fair bit).

**Comment by ernestyao** [comment]
Yes. I agreee. So I think pandoc the Rmd file and create a html file would be sufficient. In the html file, LaTeX parts can be rewriten online and then export or printed to PDF by Safari.
